recipe/meta.yaml

```
@@ -30,14 +26,17 @@ requirements:
 
   run:
     - python >=3.6
-    - setuptools
     - clyent >=1.2.0
+    - conda-package-handling >=1.7.3
     - nbformat >=4.4.0
     - pyyaml >=3.12
     - python-dateutil >=2.6.1
     - pytz
     - requests >=2.9.1
```

Looks like [`requests` got bumped to `2.20.0`]( https://github.com/Anaconda-Platform/anaconda-client/blob/2893410fa23ccd14e514c945a1f0e744adc7cb83/requirements.txt#L6 )

```suggestion
    - requests >=2.20.0
```

Addressing in PR ( https://github.com/conda-forge/anaconda-client-feedstock/pull/35 )

---

Submitted PR ( https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/355 ) to hot-fix the repodata as well